lib/open_telemetry.ex

```diff
@@ -141,10 +141,22 @@ defmodule OpenTelemetry do
   @spec link(trace_id(), span_id(), attributes(), tracestate()) :: link()
   defdelegate link(trace_id, span_id, attributes, tracestate), to: :opentelemetry
 
+  @doc """
+  Creates a `t:link/0` from a `t:span_ctx/0`.
+  """
+  @spec link(span_ctx() | :undefined) :: link()
+  defdelegate link(span_ctx), to: :opentelemetry
+
+  @doc """
+  Creates a `t:link/0` from a `t:span_ctx/0` and list of `t:attributes/0`.
+  """
+  @spec link(span_ctx() | :undefined, attributes()) :: link()
+  defdelegate link(span_ctx, attributes), to: :opentelemetry
+
   @doc """
   Creates a list of `t:link/0` from a list of 4-tuples.
   """
-  @spec links([{integer(), integer(), list(), list()}]) :: [link()]
+  @spec links([{integer(), integer(), attributes(), tracestate()} | {span_ctx(), attributes()}]) :: [link()]
```

Comment:
Spec is missing the case of an `span_ctx()` alone, covered in https://github.com/open-telemetry/opentelemetry-erlang-api/pull/69/files#diff-e77219541243e32f9b058272172d5b17R205

```suggestion
  @spec links([{integer(), integer(), attributes(), tracestate()} | span_ctx() | {span_ctx(), attributes()}]) :: [link()]
```